apps/meteor/app/models/server/raw/VideoConference.ts

```diff
@@ -1,9 +1,18 @@
 import { IVideoConference } from '@rocket.chat/core-typings';
+import { UpdateOneOptions, UpdateQuery, UpdateWriteOpResult } from 'mongodb';
 
 import { BaseRaw, IndexSpecification } from './BaseRaw';
 
 export class VideoConferenceRaw extends BaseRaw<IVideoConference> {
 	protected modelIndexes(): IndexSpecification[] {
 		return [{ key: { rid: 1, status: 1, createdAt: 1 }, unique: false }];
 	}
+
+	public updateOneById(
+		_id: string,
+		update: UpdateQuery<IVideoConference> | Partial<IVideoConference>,
+		options?: UpdateOneOptions & { multi?: boolean },
```

Comment:
Update One should not have a `multi` option, it is intended to update just **one** record..